claude.md

```
@@ -1,128 +1,106 @@
 ```bash
-npm run dev     # Development server (localhost:3000)
+npm run dev     # Development (localhost:3000)
 npm run build   # Production build
 npm run lint    # ESLint
+npm run test    # Playwright tests
 ```
 
-## Project Structure
+## Architecture Summary
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)